File: dehaze-java/src/main/java/com/pei/dehaze/model/dto/ImageFileInfo.java
package com.pei.dehaze.model.dto;

import io.swagger.v3.oas.annotations.media.Schema;
import lombok.Builder;
import lombok.Data;

@Data
@Builder
public class ImageFileInfo {

    @Schema(description = "当前图片id")
    private Long id;

    @Schema(description = "所属数据项id")
    private Long datasetItemId;

    @Schema(description = "所属文件id")
    private Long fileId;

    @Schema(description = "图片类型")
    private String type;

    @Schema(description = "文件描述")
    private String description;

    @Schema(description = "文件URL")
    private String url;
}

File: dehaze-go/model/dto/image_file_info.go
package dto

// ImageFileInfo 图片文件信息
type ImageFileInfo struct {
	// 当前图片id
	ID int64 `json:"id"`
	// 所属数据项id
	DatasetItemID int64 `json:"datasetItemId"`
	// 所属文件id
	FileID int64 `json:"fileId"`
	// 图片类型
	Type string `json:"type"`
	// 文件描述
	Description string `json:"description"`
	// 文件URL
	URL string `json:"url"`
}
